components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -78,12 +78,60 @@ impl Add for MigrationPhaseMetrics {
     }
 }
 
-pub fn migrate_sqlcipher_db_to_plaintext(
+pub fn migrate_logins(
+    path: impl AsRef<Path>,
+    new_encryption_key: &str,
+    sqlcipher_path: impl AsRef<Path>,
+    sqlcipher_key: &str,
+    salt: Option<String>,
```

Comment:
Could you keep a note about the `salt` argument -- even just the small one from before?  I don't feel like I'm going to remember how/when that was used once this project is in the past.